github/src/main/kotlin/gropius/sync/github/NodeSourcerer.kt

```diff
@@ -100,8 +101,8 @@ class NodeSourcerer(
      * @param info The issue to created this body for
      * @return The gropius issue and the mongodb issue mapping
      */
-    suspend fun ensureIssue(info: IssueData): Pair<Issue, IssueInfo> {
-        val issueInfo = issueInfoRepository.findByGithubId(info.id)
+    suspend fun ensureIssue(imsProject: IMSProject, info: IssueData): Pair<Issue, IssueInfo> {
```

Comment:
Param doc for `imsProject`

Reply:
Already modified in future PR, will not risk Merge Conflicts